biosignals.py

```diff
@@ -0,0 +1,74 @@
+import platform
+import sys
+from dataclasses import dataclass, field
+
+import plux
+
+osDic = {"Darwin": "MacOS",
+         "Linux": "Linux64",
+         "Windows": ("Win32_37", "Win64_37")}
+if platform.system() != "Windows":
+    sys.path.append("PLUX-API-Python3/{}/plux.so".format(osDic[platform.system()]))
+else:
+    if platform.architecture()[0] == '64bit':
+        sys.path.append("PLUX-API-Python3/Win64_37")
+    else:
+        sys.path.append("PLUX-API-Python3/Win32_37")
+
+
+@dataclass
+class Device:
+    """
+    Stores the integer corresponding to the port occupied by the device.
+    Defaults to port 1
+    """
+
+    source: str
+
+    def set_port(self, source: int):
+        self.source = source
+
+
+@dataclass
+class Session(Device):
+    """
+    Stores session data in a dict
+    """
+    Device: Device
+    Session_Data: dict
```

Comment:
Apologies for sounding like a broken record, but python coding standards is that arguments / parameters / variables / class methods should be lower_snake_case.

I do make a similar mistake in my first python project because my background is PHP / Java etc and that is lowerCamelCase.

Reply:
do I also have to do that for the classes I reference? cause Device is referencing the Device class, which holds the information on which port is is at and how many channels I am using

Reply:
Class names can be camel case (i.e. Device, SessionData). References to the classes elsewhere should be lower snake case i.e. device and session_data. That's because these are arguments / parameters or variable names not actual classes.

Take a look at the PEP 8 style guide for more info https://www.python.org/dev/peps/pep-0008/

Reply:
I may have misunderstood you, with the above example you would do the following

```python
device: Device
session_data: dict
```